runelite-client/src/main/java/net/runelite/client/plugins/timers/GameTimer.java

```diff
@@ -92,28 +99,69 @@
 		this.duration = Duration.of(time, unit);
 	}
 
+	GameTimer(int itemId, int quantity, String description, Integer graphicId, long time, ChronoUnit unit)
```

Comment:
shouldnt this quantity be just always 1?

Reply:
I guess? I wrote this constructor with two reasons in mind:

1. The item and sprite constructors can't both have the same arguments (if there's a better way to handle that, I'm all ears)
2. It's unlikely, but possible someone might want to have a timer with a stackable item as the displayed image, like a pile of coins.